docs/_static/custom.css

```diff
@@ -1,3 +0,0 @@
-body div.sphinxsidebarwrapper p.logo {
```

Comment:
This was made for alabaster theme, I don't think we need it now.

Reply:
I checked this and it doesn't make difference, as far as I see.

Reply:
Yes, this worked only for alabaster.
https://github.com/bitprophet/alabaster/blob/master/alabaster/static/alabaster.css_t#L141

If someone wants to implement this for the new theme, I would gladly accept the PR, but it's better not to have code which doesn't do what it looks like.